components/form/formitems/overview.md

```diff
@@ -75,12 +76,40 @@ The `FormItem` tag exposes the following parameters which you can use to customi
 }
 ````
 
->caption The result from the code snippet above
+## Hybrid Form - Combination of manually defined and autogenerated fields
```

Comment:
The heading style must match the rest of the headings in the article. In this article, we see different styles:
**Basics**
**Customize the appearance of the editors in the Form**
**Hybrid Form - Combination of manually defined and autogenerated fields**

* To match the style of the second heading in this article, you can apply one of the suggestions provided by Dimo or go with **Combine Manually Defined and Autogenerated Fields**.
* You can also fix the [capitalization ](https://docs.telerik.com/style-guide/titles-and-headings#capitalization)of the second heading: **Customize the Appearance of the Editors in the Form**

```suggestion
## Combine Manually Defined and Autogenerated Fields
```